### ISSUE TITLE
Boolean 'handled' value is not deprecated

### ISSUE THREAD

**Issue by juliankrispel** [facebook/draft-js#1332]
[The 0.9 change log](https://github.com/facebook/draft-js/releases/tag/v0.9.0) says  that boolean values are deprecated for `handle` commands. However, the [isEventHandled method](https://github.com/facebook/draft-js/blob/master/src/component/utils/isEventHandled.js) still accepts booleans.

**Comment by icd2k3** [comment]
Also worth noting parts of the docs still reference boolean values. Ex [here](https://draftjs.org/docs/api-reference-editor.html#handlepastedtext):

> Handle text and html(for rich text) that has been pasted directly into the editor. Returning **true** will prevent the default paste behavior.

**Comment by flarnie** [comment]
Good catch - this could have been done better and we would love your help in cleaning it up.

Ideally we follow a similar strategy to React regarding deprecations:

> When we add a deprecation warning, we keep it for the rest of the current major version, and change the behavior in the next major version.

(from https://facebook.github.io/react/contributing/design-principles.html#stability)

Since Draft hasn't reached 1.0, I think of 0.11.0 as the next "major" and 0.10.2 as the next "minor" release.

I think we could add a deprecation warning for passing a boolean and release that in 0.10.2, update the docs, and then completely remove the support for passing a boolean in 0.11.0.

What do you think?

**Comment by juliankrispel** [comment]
:+1: I'll update the pr

**Comment by juliankrispel** [comment]
@flarnie right, I created an issue as well [to remind us to do that](https://github.com/facebook/draft-js/issues/1336) would be good to have an appropriate label on that. 

@flarnie I guess I could just open a pr against the 0.11-alpha branch?
